package.json

```diff
@@ -34,7 +35,6 @@
     ]
   },
   "devDependencies": {
-    "react-bootstrap": "^1.0.1",
-    "react-test-renderer": "^16.13.1"
+    "react-bootstrap": "^1.0.1"
```

Comment:
Do we need this?

Reply:
bootstrap, yep

Reply:
But this commit is about testing. So I'm guessing we're not doing any styling. Is my assumption correct?

Reply:
this test is made to test Register UI which already uses react-bootstrap as dependencies. it's not added for the purpose of testing, but for creating Register form UI